nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PostHTTP.java

```diff
@@ -880,6 +936,29 @@ public long getContentLength() {
         }
     }
 
+    private FlowFile appendResponseData(final ProcessContext context, final ProcessSession session, final FlowFile original, String responseBody, int statusCode, String mimeType) {
+        FlowFile appended = original;
+        switch (context.getProperty(RESPONSE_BODY_DESTINATION).getValue()) {
+            case DESTINATION_ATTRIBUTE:
+                if (context.getProperty(RESPONSE_BODY_ATTRIBUTE).isSet()) {
+                    appended = session.putAttribute(appended, context.getProperty(RESPONSE_BODY_ATTRIBUTE).getValue(), responseBody);
+                } else {
+                    getLogger().warn("{} is set to {} but {} is not set", new Object[]{RESPONSE_BODY_DESTINATION.getName(),DESTINATION_ATTRIBUTE,RESPONSE_BODY_ATTRIBUTE.getName()});
```

Comment:
This should not be a warn. It should be an ERROR.

I think it would be a good idea to add a custom validator to identify this very same issue during configuration, instead of ignoring it during execution.

What do you think?